jwt_test.go

```diff
@@ -435,9 +435,10 @@ func TestServeHTTPAllowedByOPA(t *testing.T) {
 }
 
 func TestServeHTTPForbiddenByOPA(t *testing.T) {
+	opaResponse := "{ \"result\": { \"allow\": false, \"foo\": \"Bar\" } }"
```

Comment:
I am only trying to reuse the same variable here... so that  I can test that the response matches the input when later I passed to the validateOpaResponse method in line 498